src/compiler/transformers/static-to-meta/component.ts

```diff
@@ -55,64 +57,49 @@ export const parseStaticComponentMeta = (
   const isCollectionDependency = moduleFile.isCollectionDependency;
   const encapsulation = parseStaticEncapsulation(staticMembers);
   const cmp: d.ComponentCompilerMeta = {
```

Comment:
the diff here looks scarier than it is, I only actually changed two lines but the object wasn't alphabetized

one Q for reviewers: do we think this is safe? It's possible that some of the helper methods here have side-effects which would mean that we can't assume it's possible to safely re-order them

Reply:
I ended up backing that change out already as I was trying to debug the karma issues, pushing that code up now